databases/business-sync/fragment-shoebox-import-rules.js

```diff
@@ -0,0 +1,82 @@
+{
+  channels: getDocSyncChannels(doc, oldDoc, 'SHOEBOX_ITEMS'),
+  typeFilter: function(doc, oldDoc) {
+    // Keyspace schema:  biz.<biz_id>.shoebox.importRules
+    return createBusinessEntityRegex('shoebox.importRules').test(doc._id);
+  },
+  allowUnknownProperties: false,
+  cannotDelete: true,
+  propertyValidators: {
+    // hashtable of rules: ID -> rule
+    rules: {
+      type: 'hashtable',
+      hashtableKeysValidator: {
+        mustNotBeEmpty: true
```

Comment:
worth documenting that the id can be anything? worth limiting it to strings? what is the id used for?

Reply:
👍